ansible_collections/arista/avd/roles/eos_validate_state/anta_integration.md

```diff
@@ -211,7 +201,6 @@ only_failed_tests: "{{ validation_role.only_failed_tests | arista.avd.default(fa
 
 
 # Variable to enable ANTA eos_validate_state
-# Defaults to false as ANTA is currently in preview
```

Comment:
It seems wrong to show the value as `false` here with a description above "variable to enable ..."

Reply:
I'll work with @carl-baillargeon to improve this section as there is a mix of default variables and input variables.